server/schedulers/hot_region.go

```diff
@@ -651,6 +649,17 @@ func (bs *balanceSolver) tryAddPendingInfluence() bool {
 	return true
 }
 
+func (bs *balanceSolver) collectPendingInfluence(peer *statistics.HotPeerStat) statistics.Influence {
+	infl := statistics.Influence{Loads: make([]float64, statistics.RegionStatCount), Count: 1}
+	bs.rwTy.SetFullLoadRates(infl.Loads, peer.GetLoads())
+	inverse := bs.rwTy.Inverse()
+	another := bs.GetHotPeerStat(inverse, peer.RegionID, peer.StoreID)
```

Comment:
I prefer summary all loads at a same time (including read and write) in `prepareForBalance`, then get it from the `stLoadInfos`. it is at a same "snapshot" rather than get it one by one.

Reply:
Now we don't need to use read and write statistics together. I think we can do this when we unify read and write schedulers later.

Reply:
yes, it will be placed into together when we unify read and write schedulers.